docs/user-guide/sdh/release-hub.md

```diff
@@ -0,0 +1,265 @@
+
+---
+
+## Adding Applications
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add applications to a release track.
+{% endhint %}
+
+This involves the inclusion of applications you wish to rollout in the release version created by you.
+
+1. Click **+ Add Application** button present within the release you created.
+
+    ![Figure 8: Adding Apps to Release](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-apps.jpg)
+
+2. Click the **Search and add applications** dropdown.
+
+3. Use the checkbox to add applications from your projects.
+
+    ![Figure 9: Choosing Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/search-apps.gif)
+
+4. Click **Add Release Stage**.
+
+    ![Figure 10: Dividing Release in Stages](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-release-stage.jpg)
+
+{% hint style="info" %}
+### Importance of Release Stages
+By default, your selected applications will be set to release in one go. However, you can also release them in stages. In other words, you can decide which set of applications should be released first, subsequently which ones to release next, and the ones to release last. <br /><br /> For example, if you have a new login feature (backend) and a dashboard update (frontend), you might want to release the login feature first to ensure users can log in before seeing the updated dashboard. 
+{% endhint %}
+
+5. Use the drag-and-drop feature to move applications from one stage to another.
+
+    ![Figure 11: Rearranging the Sequence](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/release-stage-v2.gif)
+
+{% hint style="warning" %}
+The drag-and-drop feature is designed specifically for moving applications between different release stages. It is not meant to alter the sequence of applications within the same stage.
+{% endhint %}
+
+6. Once you have finalized the sequence and stages, click **Save Changes**.
+
+---
+
+## Selecting Images
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to select images for selected applications.
+{% endhint %}
+
+1. Select a workflow available for your application. All the [images](../../reference/glossary.md#image) available in the selected workflow will appear.
+
+    ![Figure 12: Selecting Image from Specific Workflow](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/workflow-selection.gif)
+
+2. Click **SELECT** next to the image you wish to deploy from the list.
+
+3. Repeat the above steps for other applications you added in the release.
+
+    ![Figure 13: Repeating Steps for Other Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/image-selections.gif)
+
+4. Click **Save**.
+
+{% hint style="info" %}
```

Comment:
We should mention that only the built images will appear in the dashboard. If there are no images present, the user will need to perform the CI stage and build the application first to obtain the image.

Reply:
Will include this in the next iteration of this doc

Reply:
Change in plans...committed in this iteration itself 👍